<!DOCTYPE HTML>
<html>
  <head>
    <title>Software</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
    <link rel="stylesheet" href="assets/css/main.css" />
    <noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
  </head>
  <body class="is-preload">
    <!-- Wrapper -->
      <div id="wrapper">

        <!-- Header -->
          <header id="header">
            <div class="inner">

              <!-- Logo -->
                <a href="index.html" class="logo">
                  <span class="symbol"><img src="images/logo.svg" alt="" /></span><span class="title">Molecular Systems Evolution</span>
                </a>

              <!-- Nav -->
                <nav>
                  <ul>
                    <li><a href="#menu">Menu</a></li>
                  </ul>
                </nav>

            </div>
          </header>

        <!-- Menu -->
          <nav id="menu">
            <h2>Menu</h2>
            <ul>
              <li><a href="index.html">Home</a></li>
              <li><a href="coevolution.html">Molecular Coevolution</a></li>
              <li><a href="popgen_modeling.html">Population Genomics Modeling</a></li>
              <li><a href="gene_networks.html">Evolutionary Models of Gene Networks</a></li>
              <li><a href="adaptive_evolution.html">Adaptive evolution</a></li>
              <li><a href="fungi_genomics.html">Population Genomics of Fungi</a></li>
              <li><a href="software.html">Data and Software</a></li>
              <li><a href="people.html">People</a></li>
            </ul>
          </nav>

        <!-- Main -->
          <div id="main">
            <div class="inner">
              <h1>Data and Software</h1>
	      <span class="image main"><img src="images/twin_trees_banner.png" alt="" /><br/>Twin trees &copy; Julien Y. Dutheil.</span>
	      <p>Scripts, data, and supplementary material of the group is available under Open Access on our <a href="https://gitlab.gwdg.de/molsysevol">GitLab repository</a>.
	      </p>

	      <h2>Software</h2>
	      
	      <h3><a href="https://biopp.github.io/">The Bio++ libraries</a></h3>
	      <p>Bio++: A C++ library for sequence analysis, phylogenetics, molecular evolution and population genetics.
	      The goal of Bio++ is to provide "bricks" for building efficient software for molecular evolution (including phylogenetics, population genetics, genomics, etc). We therefore research both on the design of the code (object orientation, ontology) and implementation (efficient algorithms and data structures).</p>

	      <h3><a href="https://jydu.github.io/sgedtools">The SGED Tools</a></h3>
	      <p>The Site/Group Extended Data format and tools, to handle sequence/alignment/3D structure annotations.
	      The SGED format is natively supported by the Bio++ libraries and derivative programs.
	      The package provides standalone Python tools to manipulate SGED files and perform statistical analyses such as (conditional) randomization.</p>

	      <h3><a href="https://github.com/gvbarroso/ismc">iSMC</a></h3>
	      <p>The iSMC framework. iSMC is developed and maintained by Gustavo Barroso.</p>
	      
	      <h3><a href="https://github.com/jydu/coalhmm">CoalHMM</a></h3>
	      <p>The CoalHMM software implements the CoalHMM model described in (Hobolth et all 2007, Dutheil et al 209, and Mailund et al 2011).</p>

	      <h3><a href="https://jydu.github.io/maffilter/">MafFilter</a></h3>
	      <p>MafFilter is a program allowing the design of advanced filtering and processing pipelines for genome alignments.</p>

	      <h3><a href="https://jydu.github.io/comap/">CoMap</a></h3>
	      <p>CoMap is a package dedicated to coevolution analysis by Cosubtitution Mapping. In contains the comap program and the MICA program (Mutual Information Coevolution Analysis), implementing several MI-based methods from the literature (Dutheil 2012, Brieffings in Bioinformatic). CoMap was developed using the Bio++ libraries.</p>
	      
	      <h3><a href="https://jydu.github.io/physamp/">PhySamp</a></h3>
	      <p>PhySamp is a package dedicated to phylogenetic sampling, that is, the filtering of sequence alignments and corresponding phylogenetic trees. It contains two programs: bppAlnOptim, which optimizes the size of an alignment while minimizing the occurrence of missing data, and bppPhySamp, which samples an alignment to minimize redundancy based on a phylogeny.</p>

	      <h3><a href="https://github.com/biopp/testnh">TestNH</a></h3>
	      <p>The TestNH package contains programs for testing and fitting non-homogeneous models of sequence evolution.The testnh program is described in Dutheil and Boussau (2008), the mapnh program is described in Romiguier et al (2012), and the partnh program in Dutheil et al (2012).</p>

	      <h3><a href="https://github.com/jydu/contest">ConTest</a></h3>
	      <p>Another program making use of substitution mapping procedures, to detect positions in a molecule under biochemical constraint (Dutheil 2007).</p>

            </div>
          </div>

        <!-- Footer -->
          <footer id="footer">
            <div class="inner">
              <ul class="copyright">
                <li>&copy; Julien Y. Dutheil. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
              </ul>
            </div>
          </footer>

      </div>

    <!-- Scripts -->
      <script src="assets/js/jquery.min.js"></script>
      <script src="assets/js/browser.min.js"></script>
      <script src="assets/js/breakpoints.min.js"></script>
      <script src="assets/js/util.js"></script>
      <script src="assets/js/main.js"></script>

  </body>
</html>
